tests/v2/validation/provisioning/rke2/provisioning_node_driver_test.go

```diff
@@ -348,6 +347,7 @@ func (r *RKE2NodeDriverProvisioningTestSuite) ProvisioningRKE2CNICluster(provide
 }
 
 func (r *RKE2NodeDriverProvisioningTestSuite) TestProvisioning() {
+	r.T().Skip()
```

Comment:
was this for debugging as well? We plan to remove this function anyway in the RKE2 refactoring so I guess it isn't that important.